iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/MergedLine.java

```diff
@@ -415,14 +415,15 @@ public String setProperty(final String key, final String value) {
         return (String) properties.setProperty(key, value);
     }
 
-    // -------------------------------
-    // Not implemented methods -------
-    // -------------------------------
     @Override
     public void remove() {
-        throw MergingView.NOT_IMPLEMENTED_EXCEPTION;
+        dl1.remove();
+        dl2.remove();
```

Comment:
No removing of MergedLine is done here, but it could done.

Today, it works as following steps :
1) MergingNetworkListener::onRemovel is called twice (for each DanglingLine).
2) MergingViewIndex::remove will be called and check if MergedLine need to be remove from cache

This allow to remove MergedLine instance from cache if one of DanglingLine is removed from a Network merged

Reply:
I have a doubt: I'm not sure the Xnode is required for dangling lines. If it's your key, you could have collision issue. For me, it's really strange to rely on events to manage the removal.

Reply:
As I said in the previous PR, the name of the XNode is not required. So I think, you should not use it as a key.